examples/index.html

```diff
@@ -55,7 +55,7 @@
             .example-form > * {
                 max-width: 300px;
             }
-            .example-code textarea {
+            .example-code pre {
```

Comment:
Why the change to `pre`s? `textarea`s allow user focus, so one can easily select the entire element’s contents.